blog.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>>Adopt A Dog</title>
+
+  <link rel="stylesheet" type="text/css" href="./styles/blog.css">
+
+</head>
+
+<body>
+  <header>
+    <div class="row">
+      <div class="col span_4_of_12">
+        <img class="header-logo" src="./images/adoptadog-logo.svg" alt="adopt a dog logo">
+        <h1>Seattle's Premiere dog adoption agency</h1>
+
+      </div>
+      <div class="col span_7_of_12">
```

Comment:
If you're going to write your own grid, see if you can write an offset that you could apply to the next column instead of an extra grid.

That being said you could layout  this whole header with flexbox